### PR TITLE
[OSDEV-939] Facility Claim - UI fixes

### PR DESCRIPTION
**Pull request by roman-stolar**
[OSDEV-939](https://opensupplyhub.atlassian.net/browse/OSDEV-939) **Facility Claim - UI fixes**
- Fixed Mariel UI comment -> https://opensupplyhub.atlassian.net/browse/OSDEV-939?focusedCommentId=12580
- Fix Field Validation

### CODE REVIEW

**Review by mazursasha1990 — APPROVED** [review]
LGTM

File: src/django/api/views/facility/facilities_view_set.py [hunk 1]
```diff
@@ -904,9 +904,8 @@ def claim(self, request, pk=None):
 
             facility_claim.facility_workers_count = workers_count
 
-            if local_language_name != '':
-                facility_claim.\
-                    facility_name_native_language = local_language_name,
+            facility_claim.\
```
Comment (VadimKovalenkoSNF) [inline src/django/api/views/facility/facilities_view_set.py:907]: Why did you remove this condition?

File: src/react/src/util/util.js [hunk 2]
```diff
@@ -908,10 +908,15 @@ export const claimAFacilityFormIsValid = ({
         isEmpty(numberOfWorkers),
         NUMERIC_DASH_REGEX.test(numberOfWorkers),
     ]) &&
-    some([
-        !isEmpty(businessWebsite) && isURL(businessWebsite),
-        !isEmpty(businessLinkedinProfile) && isURL(businessLinkedinProfile),
-        !isEmpty(businessUploadFiles),
+    every([
+        isEmpty(businessWebsite) ||
```
Comment (VadimKovalenkoSNF) [inline src/react/src/util/util.js:912]: It's hard to understand the logic behind these conditions with multiple `||` and `&&` operators. Could you refactor this by replacing statements in the parenthesis into separate functions?

**Review by VadimKovalenkoSNF — CHANGES_REQUESTED** [review]

File: src/django/api/views/facility/facilities_view_set.py (hunk 1 above)
Comment (roman-stolar) [reply src/django/api/views/facility/facilities_view_set.py:907]: It’s useless + I remove comma in the end that bring issue. @VadimKovalenkoSNF

Comment (VadimKovalenkoSNF) [reply src/django/api/views/facility/facilities_view_set.py:907]: Ack

File: src/react/src/util/util.js (hunk 2 above)
Comment (roman-stolar) [reply src/react/src/util/util.js:912]: I suppose to refactor it later on, as it didn’t bring any risk at the moment, only a visibility to read. @VadimKovalenkoSNF

**Review by VadimKovalenkoSNF — APPROVED** [review]
Preliminary approved.
